implementations/index.js

```diff
@@ -34,6 +34,14 @@ manifests.local = [
   'localImplementationsConfig.cjs'
 ].flatMap(path => getLocalManifest(path));
 
+// get localConfig and look for a manifest
+const localConfig = [
+  '.localConfig.cjs',
+  'localConfig.cjs'
```

Comment:
Let's pick one or the other. Supporting both is just another opportunity for confusion/error.

@aljones15 @tminard thoughts on which you'd like better?

Reply:
We agreed that `localConfig.cjs` was best.